ikasaneip/sample/spring-boot/jms/src/test/java/com/ikasan/sample/spring/boot/builderpattern/JmsSampleFlowTest.java

```diff
@@ -155,42 +162,39 @@
 
     private IkasanFlowTestRule flowTestRule;
 
-    private MessageListenerVerifier messageListenerVerifier;
+    @Resource
+    @Autowired
+    @Qualifier("ikasan.xads")
+    private DataSource ikasanxads;
```

Comment:
Person writing uinttest should need to know anything about wiring ikasanDatasouce

Reply:
See previous - this isnt the final framework